specs-go/config.go

```diff
@@ -529,7 +539,28 @@ type LinuxSeccompArg struct {
 
 // LinuxSyscall is used to match a syscall in Seccomp
 type LinuxSyscall struct {
-	Name   string             `json:"name"`
-	Action LinuxSeccompAction `json:"action"`
-	Args   []LinuxSeccompArg  `json:"args,omitempty"`
+	Name     string            `json:"name,omitempty"`
+	Names    []string          `json:"names,omitempty"`
+	Action   Action            `json:"action"`
+	Args     []LinuxSyscallArg `json:"args"`
+	Comment  string            `json:"comment"`
+	Includes Filter            `json:"includes"`
```

Comment:
@justincormack Not sure what you mean by upstream. Wouldn't be want filters that disable syscalls only for certain arguments?